R/version.R

```diff
@@ -240,6 +238,19 @@ format.version_sentinel <-
 .get_R_version <- function()
     getRversion()
 
+.get_sys_file <- function()
+    system.file(package = "BiocVersion")
+
+.get_BiocVersion_version <-
```

Comment:
One minor thing about these functions is the convention that functions in each file follow the naming convention `.<filename>_*`, so `.version_BiocVersion()` and `.version_BiocVersion_installed()`